services/generic/contact_sensor.md

```diff
@@ -1,14 +1,34 @@
-### Contact sensor service
+# Contact Sensor Service
 
-Binary contact sensor, normally magnetic contact.
+Contact sensor service is a binary sensor used by devices such as magnetic contact sensors usually installed in windows or doors.
 
-#### Service names
+## Service name
 
 `sensor_contact`
 
-#### Interfaces
+## Interfaces
 
-Type | Interface           | Value type | Description
------|---------------------|------------|--------------------
-in   | cmd.open.get_report | null       |
-out  | evt.open.report     | bool       | true = open
\ No newline at end of file
+| Type | Interface           | Value type | Description                                       |
+|------|---------------------|------------|---------------------------------------------------|
+| in   | cmd.open.get_report | null       | Requests the contact state report.                |
```

Comment:
Request issue of 'evt.open.report' response -> Maybe we could follow such pattern everywhere ?

Reply:
I think it is within the realm of self-evident. With adapter guidelines that are partially now in Notion, but will be in the future in fimp-api as well, there should be no issue with this. Please bear in mind that before Description column was usually empty at all.